src/midlewares/types.ts

```diff
@@ -0,0 +1,7 @@
+import * as express from "express";
```

Comment:
have you checked if this type exists into @types/express and is something like express.Hander ?